app/Resources/views/default/sponsor.html.twig

```diff
@@ -7,7 +7,7 @@
 
 {% block body %}
     <section class="page-content clearfix sponsor-index ">
-        <h1>2016 South Dakota Code Camp Sponsors</h1>
+        <h1>2018 South Dakota Code Camp Sponsors</h1>
```

Comment:
we'll need to remove all the sponsors on this page first.

Reply:
I updated the file

Reply:
@harmstyler The speakers and schedules need to be emptied.

I removed the family from the menu

I'm also not finding the register page in the code. I think we'd just redirect it to the EventBrite page?